alibi_detect/saving/loading.py

```diff
@@ -253,7 +253,10 @@ def _load_preprocess_config(cfg: dict) -> Optional[Callable]:
         logger.warning('Unable to process preprocess_fn. No preprocessing function is defined.')
         return None
 
-    return partial(preprocess_fn, **kwargs)
+    if kwargs == {}:
+        return preprocess_fn
```

Comment:
Previously the preprocess function was wrapped in a `partial` even if there were no kwarg's. Now the bare function is returned in this case.